tests/test_prompt_chatbot.py

```diff
@@ -150,3 +150,53 @@ def test_missing_persona_fails(mock_co: object) -> None:
     """
     with pytest.raises(FileNotFoundError):
         _ = PromptChatbot.from_persona("invalid_persona", mock_co)
+
+
+def test_update_max_context_fails(
+    mock_prompt_chatbot: PromptChatbot, mock_co: object
+) -> None:
+    """Tests failure on updating max_context_examples
+
+    prompt_size with more than 2048 tokens, even changing max_context_size
+
+    Args:
+        mock_prompt_chatbot (PromptChatbot): Bot test fixture
+        mock_co (object): mock Cohere client.
+
+    """
+    with pytest.raises(ValueError):
+        current_prompt = mock_prompt_chatbot.get_current_prompt(query=f"{'a'*2048}")
+        max_context_examples = 10
+        prompt_size = len(mock_co.tokenize(current_prompt))
+        mock_prompt_chatbot._update_max_context_examples(
```

Comment:
I checked this on local: I'm concerned that this test always passes because `mock_prompt_chatbot.prompt_size_history` is an empty list here, so the ValueError is always raised

Reply:
I didn't thought about that! Thanks!! I'll try to solve and think about the best way to test this function